src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -141,9 +146,49 @@ public IActionResult LaunchDebugger()
                 : StatusCode(StatusCodes.Status500InternalServerError, new { status = error });
         }
 
+        [HttpPost]
+        [Route("admin/host/restart")]
+        //[Authorize(Policy = PolicyNames.AdminAuthLevel)]
+        public IActionResult Restart()
+        {
+            _scriptHostManager.RestartHost();
+            return Ok(_webHostSettings);
+        }
+
+
+        [HttpGet]
+        [Route("admin/host/download")]
+        //[Authorize(Policy = PolicyNames.AdminAuthLevel)]
+        public IActionResult Download()
```

Comment:
I wonder if this doesn't belong on FunctionsController where we're already doing List operations? E.g. admin/functions/download makes sense - download all my functions. That makes more sense than admin/host/download I think.

Reply:
good catch, it does make more sense in the FunctionsController. Will move it there